python/cudf/cudf/tests/test_groupby.py

```diff
@@ -1666,3 +1666,17 @@ def test_groupby_mix_agg_scan():
     gb.agg(func[1:])
     with pytest.raises(NotImplementedError, match=err_msg):
         gb.agg(func)
+
+
+@pytest.mark.parametrize(
+    "data", [{"Speed": [380.0, 370.0, 24.0, 26.0], "Score": [50, 30, 90, 80]}],
+)
+@pytest.mark.parametrize("group", ["Score", "Speed"])
+def test_groupby_describe(data, group):
+    pdf = pd.DataFrame(data)
+    gdf = cudf.from_pandas(pdf)
+
+    got = gdf.groupby(group).describe()
+    expect = pdf.groupby(group).describe()
+
+    assert_groupby_results_equal(expect, got, check_dtype=False)
```

Comment:
Am I missing something here...

https://github.com/rapidsai/cudf/blob/eb88a388f6fd5fe5d983d74adbfda78a4251974e/python/cudf/cudf/tests/test_groupby.py#L33-L43

It looks like this function only tests equality when `sort=False`... Is it broken? It's used many places in `test_groupby.py`.

Reply:
fix is introduced here: https://github.com/rapidsai/cudf/pull/8272/

let's incorporate these changes before proceeding.

Reply:
However, since Pandas returns a `MultiIndex Column `whereas cudf returns `Index Column`, how do i compare the results?
Or do i have to convert cudf into a MultiIndex too?